styles.css

```diff
@@ -8,3 +8,53 @@ red - #ee5253
 
 */
 
+* {
+  border: solid red 2px;
+  font-family: 'Roboto', sans-serif;
+}
+
+.top-page {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+body, html {
+  height: 100%;
+}
+
+.background-container {
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), var("assets/burger-friends.jpg");
+  background-size: cover;
+  height: 100%;
+  width: 100%;
+}
+
```

Comment:
Need to understand background image and how to overlay with other HTML elements. Researching that now.